sesman/chansrv/chansrv_fuse.c

```diff
@@ -443,8 +443,11 @@ xfuse_init(void)
     }
 
     /* setup xrdp file system */
-    if (xfuse_init_xrdp_fs())
-        return -1;
+    if (g_xrdp_fs.inode_table == NULL)
+    {
+        if (xfuse_init_xrdp_fs())
+            return -1;
```

Comment:
Always use braces for newly touched codes. The exiting code is not always good example. The original code violates our coding standard.